vite.config.mjs

```diff
@@ -1,7 +1,41 @@
 import vue2 from "@vitejs/plugin-vue2";
 import { defineConfig } from "vite";
 import laravel from "laravel-vite-plugin";
-import manifestSRI from "vite-plugin-manifest-sri";
+import { createHash } from "node:crypto";
+import { resolve } from "node:path";
+import { readFileSync, writeFileSync } from "node:fs";
+
+function manifestQueryParam() {
```

Comment:
This vite plugin aims to provide the same functionality of cache busting (with md5 hash route parameter) as laravel mix did instead of using integrity checks on the tags and passing the integrity key in the manifest.